android-example/.cargo/config

```diff
@@ -2,6 +2,6 @@
 linker = "aarch64-linux-android21-clang++"
 runner = "./run-on-android.sh"
 
-[target.arm-linux-androideabi]
+[target.armv7-linux-androideabi]
```

Comment:
I suppose app/build.gradle should be also changed, because of it has such lines:
```groovy
    'armeabi': 'arm-linux-androideabi',
```